package.json

```diff
@@ -177,7 +177,6 @@
     "@swc/helpers": "0.5.13",
     "@trivago/prettier-plugin-sort-imports": "4.3.0",
     "@types/cross-spawn": "6.0.6",
-    "@types/dompurify": "3.0.5",
```

Comment:
`dompurify@3.2.0` added built in types and so the maintainers of `@types/dompurify` have deprecated the package in favor of the built in types.